runtime-display/vulkan-utility-libraries/spec

```diff
@@ -0,0 +1,3 @@
+VER=1.3.269
+SRCS="tbl::https://github.com/KhronosGroup/Vulkan-Utility-Libraries/archive/refs/tags/v$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.